README_SChinese.md

```diff
@@ -0,0 +1,899 @@
+# Town Of Host（房主小镇）
+
+[![TownOfHost-Title](./Images/TownOfHost-Title.png)](https://youtu.be/IGguGyq_F-c)
+
+<p align="center"><a href="https://github.com/tukasa0001/TownOfHost/releases/"><img src="https://badgen.net/github/release/tukasa0001/TownOfHost"></a></p>
+
+本自述文件由两个少年汉化组进行翻译
```

Comment:
Sorry, below this sentence, "The developer group will not update this file and is not responsible for any delays in information." Please include a statement in Chinese stating that